runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/renderers/GatewayFormRenderer.tsx

```diff
@@ -106,6 +106,12 @@ const GatewayFormRenderer = () => {
         BpmnConnectionFactory.from(modeler).setConnectionColor(flow, theme.palette.common.black);
     };
 
+    const isSequenceWithoutExpression = (outgoing: IBpmnConnection) => {
+        return gateway.businessObject.default.id === outgoing.id || outgoing.businessObject.conditionExpression?.body;
```

Comment:
wrap 2nd part of the condition with `Boolean()` to have return type `boolean` not `true | unknown`